TemporaryProjects/NewTempProjectCommandPackage.cs

```diff
@@ -10,6 +11,7 @@ namespace TemporaryProjects
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [InstalledProductRegistration("#110", "#112", "1.0", IconResourceID = 400)] // Info on this package for Help/About
     [ProvideMenuResource("Menus.ctmenu", 1)]
+    [ProvideAutoLoad(VSConstants.UICONTEXT.ShellInitialized_string, PackageAutoLoadFlags.BackgroundLoad)]
```

Comment:
It looks like `ProvideCodeContainerProviderAttribute` only exists in Microsoft.VisualStudio.Shell.15.0 :/ so we can't use it if we're targeting 2015

Reply:
Or should I create a second project, which would only target VS 2017? This functionality won't work in 2015 anyway because the Start Page is different (and I don't have 2015 installed would prefer not doing that).

Reply:
> Or should I create a second project, which would only target VS 2017?

I think that's maybe a good idea. Being able to directly reference the `Shell.15.0` assemblies would clean up some other stuff as well.

Would you like create a `TemporaryProjects.15.0` project that references `Microsoft.VisualStudio.Shell.15.0`? We can then conditionally include this package in the `source.extension.vsixmanifest` dependencies.

Reply:
Ok, I've never done and don't know what to do in `vsixmanifest` so I might have to read some docs first.

Reply:
Sorry, I don't think I'll be able to do this. I have no way of testing whether my changes work because I don't have 2015 and would prefer not installing that.

Reply:
No problem.  I'll have a go at setting up the solution so that it's easy to work with when only VS 2017 is installed. I'm hoping this scenario is supported. 🤞